msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1058,7 +1027,7 @@ private void validateSilentParameters(
     }
 
     @Override
-    public void acquireTokenSilent(
+    public void acquireTokenSilentAsync(
             @NonNull final AcquireTokenSilentParameters acquireTokenSilentParameters) {
         validateSilentParameters(acquireTokenSilentParameters);
```

Comment:
There's validateAcquireTokenSilentParameters() down there. Is this (or the other) redundant?

Reply:
I'll check.

Reply:
Going to follow up on this in a future PR.